stdlib/public/Platform/winsdk.modulemap

```diff
@@ -244,6 +237,20 @@ module WinSDK [system] {
     link "shcore.lib"
   }
 
+  module System {
+    module DbgHelp {
+      header "DbgHelp.h"
+      export *
+
+      link "DbgHelp.Lib"
+    }
+
+    module IOCtl {
```

Comment:
Mind naming this `IOCTL` to match https://docs.microsoft.com/en-us/windows/win32/api/winioctl/ ?

Reply:
Is there some general rule for naming these modules? I imagine me writing something in Swift and thinking "Ok, MSDN says that these API are defined in winioctl.h, but what import do I need to add?"
At this point it looks easier to just look into modulemap and search for corresponding module.
I am torn between Swiftified module names and "platformish" MSDN-inspired names.

Reply:
@lxbndr
From the compilation POV it doesn't really matter which import you add, since all of the submodules are represented by a single underlying Clang module, so importing any submodule (or the entire `WinSDK`) implicitly imports them all.

From the convenience POV when writing the imports, as you've pointed out, looking for a specific submodule names might be frustrating, if you prefer them over importing the entire module. Ideally your IDE or editor would automatically insert the desired import depending on your preference though, and you wouldn't have to remember the submodule names.
When reading the imports, especially by someone who isn't an expert in the Windows API naming & structure, Swifty names IMO would be easier to understand than some cryptic Windows abbreviations.

There was also a related discussion in https://github.com/apple/swift/pull/33929, where @compnerd was in favor of the MSDN section names, which don't usually match the header names, and are closer to the Swifty names.

Reply:
@egorzhdan Oh, thanks for pointing on that discussion. Compnerd's suggestion has sense.